comp/core/workloadmeta/collectors/internal/podman/podman.go

```diff
@@ -63,7 +63,11 @@ func (c *collector) Start(_ context.Context, store workloadmeta.Component) error
 		return dderrors.NewDisabled(componentName, "Podman not detected")
 	}
 
-	c.client = podman.NewDBClient(config.Datadog.GetString("podman_db_path"))
+	if !config.Datadog.GetBool("podman_use_sqlite") {
```

Comment:
Can we implement auto detection instead if we mount `/var/lib/containers/storage`